spec/fizz_buzz_spec.rb

```diff
@@ -1 +1,48 @@
 require "fizz_buzz"
+
+describe FizzBuzz do
+
+  describe ".divisible_by" do
+
+    context "given number, returns number" do
+      it "one returns one" do
```

Comment:
doesn't read like a sentence: It one returns one.  Better would be?  It returns one given one.

Reply:
Same comment for all test names..